src/docs/development/data-and-backend/json.md

```diff
@@ -78,10 +78,13 @@ encoding, see
 
 The simple answer is no.
 
-Such a library would require using runtime reflection, which is disabled in
-Flutter. Runtime reflection interferes with _tree shaking_, which Dart has
+Such a library would require using runtime 
+[reflection](https://en.wikipedia.org/wiki/Reflection_(computer_programming)) 
+which is disabled in
+Flutter. Runtime reflection interferes with 
+[tree shaking](https://en.wikipedia.org/wiki/Tree_shaking), which Dart has
 supported for quite a long time. With tree shaking, you can “shake off” unused
-code from your release builds. This optimizes the app's size significantly.
+code from the release builds and this optimizes the app's size significantly.
```

Comment:
@kwalrath - are you ok with the changes to this paragraph?

Reply:
I like the original text better.

Reply:
How about the added links to Wikipedia?

Reply:
Ah, missed those. I like them, but we should move the punctuation inside the brackets, so the external link markers don't look weird. So:

[reflection] -> [reflection,]
[tree shaking](...), -> [treeshaking,](...)

Reply:
Also, move the link definitions to the end of the file, rather than inlining them. E.g.,

    Such a library would require using runtime [reflection][] ...
    
    [reflection]: https://en.wikipedia.org/wiki/Reflection_(computer_programming)

Reply:
Of course, if you have the commas in there, you'll probably want to use something like `[reflection,][reflection]`.

Reply:
Oops, yes, thanks.